Dev/Install/install.php5

```diff
@@ -152,7 +152,7 @@ if(isset($_REQUEST['db'])) {
 		$databaseConfig['type'] = $type;
 	}
 } else {
-	$type = $_REQUEST['db']['type'] = defined('SS_DATABASE_CLASS') ? SS_DATABASE_CLASS : 'MySQLDatabase';
+	$type = $_REQUEST['db']['type'] = defined('SS_DATABASE_CLASS') ? SS_DATABASE_CLASS : 'MySQLPDODatabase';
```

Comment:
'twas staring us in the face:

https://github.com/silverstripe/silverstripe-framework/blob/master/_register_database.php#L32